src/main/java/org/example/Main.java

```diff
@@ -13,6 +14,13 @@ public static String reverseChars(String word) {
         return String.valueOf(revChars);
     }
 
+    public static String reverseCharsStream(String word) {
+
+        return Stream.of(word.split(""))
+                .reduce("", (acc, e) -> e+acc);
```

Comment:
What about using StringBuilder?

Reply:
I didn't think about that. Is there any benefit in using string builder?